static/app/components/tours/components.tsx

```diff
@@ -292,12 +319,6 @@ export function TourGuide({
     offset,
   });
 
-  useEffect(() => {
```

Comment:
These events are now being collected in the parent TourElement component since that has access to the `tour_key`